types/16bars/Feedback.ts

```diff
@@ -0,0 +1,12 @@
+export type FeedbackType = 'timestamp' | 'general' | 'structured';
```

Comment:
Lets move these to `api` as well?

Reply:
hmm i was thinking we could just keep all the types in types/16bars? although it might make sense to specifically scope some types to `api`

Reply:
but regardless, prob good to wait on that - itll be easier to decide that structure when we setup basic endpoints imo